### PR TITLE
do not return payload as field

### PR DESCRIPTION
**Pull request by MeirShpilraien**
- [ ] cherry pick to 2.0

### CODE REVIEW

**Review by filipecosta90 — DISMISSED** [review]

**Comment by MeirShpilraien** [comment]
Tests will pass when this will be merged:
https://github.com/RedisLabsModules/RLTest/pull/104

**Review by ashtul — APPROVED** [review]

**Review by rafie — APPROVED** [review]

File: tests/pytests/test_followhashes.py [hunk 1]
```diff
@@ -207,10 +207,10 @@ def testPayload(env):
     for _ in env.retry_with_rdb_reload():
         waitForIndex(env, 'things')
         res = env.cmd('ft.search', 'things', 'foo')
```
Comment (ashtul) [inline tests/pytests/test_followhashes.py:209]: We should add 'WITHPAYLOAD`

File: tests/pytests/test_followhashes.py [hunk 2]
```diff
@@ -207,10 +207,10 @@ def testPayload(env):
...
-        env.assertEqual(toSortedFlatList(res), toSortedFlatList([1L, 'thing:foo', ['name', 'foo', 'payload', 'stuff']]))
+        env.assertEqual(toSortedFlatList(res), toSortedFlatList([1L, 'thing:foo', ['name', 'foo']]))
 
         res = env.cmd('ft.search', 'things', 'foo', 'withpayloads')
```
Comment (ashtul) [inline tests/pytests/test_followhashes.py:212]: same

File: tests/pytests/test_followhashes.py [hunk 3]
```diff
@@ -223,10 +223,10 @@ def testBinaryPayload(env):
     for _ in env.retry_with_rdb_reload():
         waitForIndex(env, 'things')
         res = env.cmd('ft.search', 'things', 'foo')
-        env.assertEqual(toSortedFlatList(res), toSortedFlatList([1L, 'thing:foo', ['name', 'foo', 'payload', '\x00\xAB\x20']]))
```
Comment (ashtul) [inline tests/pytests/test_followhashes.py:226]: same

File: tests/pytests/test_followhashes.py [hunk 4]
```diff
@@ -223,10 +223,10 @@ def testBinaryPayload(env):
...
+        env.assertEqual(toSortedFlatList(res), toSortedFlatList([1L, 'thing:foo', ['name', 'foo']]))
 
         res = env.cmd('ft.search', 'things', 'foo', 'withpayloads')
-        env.assertEqual(toSortedFlatList(res), toSortedFlatList([1L, 'thing:foo', '\x00\xAB\x20', ['name', 'foo', 'payload', '\x00\xAB\x20']]))
```
Comment (ashtul) [inline tests/pytests/test_followhashes.py:229]: same

**Review by ashtul — COMMENTED** [review]
These tests are for the payload.

File: tests/pytests/test_followhashes.py (hunk 1 above)
Comment (MeirShpilraien) [reply tests/pytests/test_followhashes.py:209]: There is withpayloads right bellow ..

File: tests/pytests/test_followhashes.py (hunk 2 above)
Comment (MeirShpilraien) [reply tests/pytests/test_followhashes.py:212]: This is withpayloads

File: tests/pytests/test_followhashes.py (hunk 3 above)
Comment (MeirShpilraien) [reply tests/pytests/test_followhashes.py:226]: same

File: tests/pytests/test_followhashes.py (hunk 4 above)
Comment (MeirShpilraien) [reply tests/pytests/test_followhashes.py:229]: same

**Review by ashtul — DISMISSED** [review]

**Comment by MeirShpilraien** [comment]
@rafie @ashtul can we merged this?

**Review by ashtul — APPROVED** [review]

**Comment by ashtul** [comment]
LGTM 👍🏻

**Pull request merged**